src/utils/version-compatible.ts

```diff
@@ -0,0 +1,17 @@
+import {Context} from '../context';
+import {ApiVersion} from '../base_types';
+
+/**
+ * Check if the current or optionally supplied version is compatible with a given version
+ */
+export default function versionCompatible(referenceVersion: ApiVersion, currentVersion: ApiVersion = Context.API_VERSION): boolean {
+  // Return true if not using a dated version
+  if (currentVersion === ApiVersion.Unstable || currentVersion === ApiVersion.Unversioned) {
+    return true;
+  }
+  const numericVersion = (version: string) => parseInt(version.replace('-', ''), 10);
+  const current = numericVersion(currentVersion);
+  const reference = numericVersion(referenceVersion);
+  console.log(current, reference, current >= reference);
```

Comment:
Seems like this was left behind :)

Reply:
I did indeed! That should have been fixed already in b87027b25a0855d59415a6ad5bf517b971a84eec.